community/muscat/workshops.en.md

```diff
@@ -15,18 +15,25 @@ To our regular Muscat users: You can also hold a Muscat workshop! Consider propo
 
 ## Upcoming workshops
 
-* Fall workshops: Muscat introduction, online, German and English.
-    - November 16 and 18, 2020 (German, English)
-    - November 23 and 25, 2020 (Italian)
-    - November 24 and 26, 2020 (English)
-    - November 29 and December 1, 2020 (English)
+Spring 2021: Dates to be announced. Write to contact@rism.info if you would like to receive an email once dates are determined.
 
-* November 2020 (Fridays): [Ewha Womans University](http://www.ewha.ac.kr/ewhaen/academics/music.do), Seoul, South Korea (JeongYoun Chang, Korea National University of Arts)
 
-* November 2020 (date TBD): Editing printed music, online (Jennifer Ward, RISM Central Office)
 
 ## Past workshops and demonstrations
 
+* Fall 2020 workshops: [Muscat introduction](http://www.rism.info/en/home/newsdetails/article/2/introductory-muscat-workshops-in-november.html), online      
```

Comment:
This link will break once we migrate the website. The link should be `/events/2020/10/22/introductory-muscat-workshops-in-november-full.html`

Reply:
Thank you! How do I go back in and make the change?

Reply:
I mean: Can I edit as normal, or do I have to go through this commit somehow?

Reply:
It's been merge, so simply edit as normal and ignore this commit to do so